src/Servers/Kestrel/tools/CodeGenerator/HttpProtocolFeatureCollection.cs

```diff
@@ -17,6 +17,7 @@ public static string GenerateFile()
                 "IServiceProvidersFeature",
                 "IHttpRequestLifetimeFeature",
                 "IHttpConnectionFeature",
+                "IHttpResponseStartFeature"
```

Comment:
Do common do we expect the usage of this feature to be? If an app uses only the response stream or always starts with PipeWriter.WriteAsync() you shouldn't need it, right? I think that this would be the majority of the cases. I recommend moving this to sometimesFeatures.